src/main/groovy/utils/BasePerson.groovy

```diff
@@ -0,0 +1,16 @@
+package com.mini.asaas.utils
+
+import com.mini.asaas.PersonType
+
+abstract class BasePerson {
+
+    String cpfCnpj
+    String name
+    String email
+    String postalCode
+    String address
+    String city
+    String state
+    PersonType personType
+
```

Comment:
Pessoa não tem telefone?
Em qual campo vamos registrar o bairro?
Em qual campo vamos registrar o número da casa?
E se for uma casa nos fundos? não tem campo para o complemento do endereço?

Senti falta das contraints dos campos tbm